src/main.js

```diff
@@ -39,22 +79,20 @@ exports.load = function(contents) {
                 traditionalPreference++
             }
 
+            const sortedEntries = sortEntries(entries);
+
             result.push({
                 text: word,
-                traditional: entries[0] ? entries[0].traditional : word,
-                simplified: entries[0] ? entries[0].simplified : word,
+                traditional: sortedEntries[0] ? sortedEntries[0].traditional : word,
+                simplified: sortedEntries[0] ? sortedEntries[0].simplified : word,
 
                 position: {
                     offset,
                     line,
                     column
                 },
 
-                matches: entries.map(({pinyin, pinyinPretty, english}) => ({
-                    pinyin,
-                    pinyinPretty,
-                    english
-                }))
+                matches: sortedEntries
```

Comment:
In the old code, we're filtering out `traditional` and `simplified` fields. I'd like to keep it that way.